en-US/Conductor_Handbook/themes/pdf-theme.yml

```diff
@@ -0,0 +1,42 @@
+extends:
+  - default
+page:
+  size: Letter
+title-page:
+  logo:
+    align: left
+    image: image:../../../assets/Free5e-Logo-Gold-Ring.png[pdfwidth=20%]
+    top: 0%
+  title:
+    font-color: #b77a2b
+    font-family: Helvetica
+    font-size: 32
+    font-style: bold
+    top: 15%
+  authors:
+    content:
+      name_only: '{author}'
+    font-color: #29326a
+    font-family: Helvetica
+    font-size: 20
+footer:
```

Comment:
Here's what the footer can look like with these settings:
<img width="686" height="886" alt="image" src="https://github.com/user-attachments/assets/d5285774-3c2d-4aa3-989b-2d6ec73dcc28" />